zeppelin-web-angular/src/app/pages/workspace/notebook/paragraph/control/control.component.html

```diff
@@ -28,22 +28,28 @@
        (click)="runParagraph.emit()"
        nzType="play-circle"
        class="run-para"
-       nzTheme="outline"></i>
+       nzTheme="outline"
+       nz-tooltip
+       nzTooltipTitle="Run paragraph"></i>
     <i nz-icon
        *ngIf="status=='RUNNING' || status=='PENDING'"
        (click)="cancelParagraph.emit()"
        class="cancel-para"
        nzType="pause-circle"
-       nzTheme="outline"></i>
+       nzTheme="outline"
+       nz-tooltip
+       nzTooltipTitle="Cancel paragraph"></i>
   </a>
-  <a (click)="toggleEditor()">
+  <a (click)="toggleEditor()" nz-tooltip nzTooltipTitle="Show/hide the code">
```

Comment:
It seems that this tooltip is covering the buttons located to the left of its target, making it difficult to click those buttons once the tooltip appears.
It might be better to fix the tooltip's position, perhaps above the target.
I think you could use a property like `nzTooltipPlacement`.

Reply:
the current behavior is ok to me given the tooltip disappears immediately after moving the mouse away

Reply:
I tried using nzTooltipPlacement="top", but it didn’t work as expected, likely due to limited space around the target element, causing the tooltip to shift to the left.

I also looked into other features like nzTooltipArrowPointAtCenter and nzTooltipOverlayStyle, but those aren't available in our current Ant Design version due to the Angular version we’re using.

As @pan3793 mentioned, I think the current behavior is acceptable. Alternatively, nzTooltipPlacement="topRight" places the tooltip above the button as in the image below, but it’s not centered, so I’d appreciate more feedback on this.
<img width="276" alt="image" src="https://github.com/user-attachments/assets/a59eb63c-ba6c-4e4d-9a8b-1b380bd04fc2">

Open to other suggestions. Thank you a lot @tbonelee

Reply:
@cherrie-k 
If the option doesn't work as expected, it's not a big deal.
The issue I mentioned was more of a secondary concern, so I'm okay with merging things as they are.
Thanks a lot for exploring alternative solutions anyway!